src/text-overflow/TextOverflow.js

```diff
@@ -41,10 +41,12 @@ export default class TextOverflow extends React.Component {
   logWarnings = () => {
     warn(() => {
       const node = React.findDOMNode(this.refs.text).parentNode.parentNode;
```

Comment:
what if `React.findDOMNode(this.refs.text)` is `null`? Are we sure this could never happen?